### PR TITLE
News page modify

### LINKED ISSUE #35: Fix Folder for news

### PR DESCRIPTION
**Pull request by GabrielAdward**

### CODE REVIEW

File: app/src/main/java/com/example/football_manager/News/CreateNews.kt [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+@Composable
+fun CreateNews(navController: NavHostController) {
+    val errors = remember { mutableStateListOf<String>() }
+    val NEWS_TITLE_MIN_LENGTH = 3
+    val NEWS_TITLE_MAX_LENGTH = 30
+    val NEWS_WRITER_NAME_MIN_LENGTH = 4
+    val NEWS_WRITER_NAME_MAX_LENGTH = 10
+    val NEWS_CONTENT_MIN_LENGTH = 10
+    val NEWS_CONTENT_MAX_LENGTH = 120
```
Comment (albinforsbergju) [inline app/src/main/java/com/example/football_manager/News/CreateNews.kt]: 120 characters is too short

File: app/src/main/java/com/example/football_manager/News/EditNews.kt [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+
+@Composable
+fun EditNews(id: Int, navController: NavHostController){
+
+    val errors = remember { mutableStateListOf<String>() }
+    val EDIT_NEWS_TITLE_MIN_LENGTH = 3
+    val EDIT_NEWS_TITLE_MAX_LENGTH = 30
+    val EDIT_NEWS_CONTENT_MIN_LENGTH = 10
+    val EDIT_NEWS_CONTENT_MAX_LENGTH = 120
```
Comment (albinforsbergju) [inline app/src/main/java/com/example/football_manager/News/EditNews.kt]: You use these constants both here and in the create news view. 
Maybe move them somewhere else so you can reuse.

File: app/src/main/java/com/example/football_manager/News/EditNews.kt [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            val updateTitleField = InputTitleTextOfNews.text
+            val updateContentField = InputContentOfNews.text
+            Spacer(modifier = Modifier.height(16.dp))
+
+            Button(
+                onClick = {
+                    errors.clear()
+                    if (updateTitleField.isEmpty() or updateContentField.isEmpty()) {
+                        errors.add("Check So No Fields Are Empty")
+                        println("Hello")
```
Comment (albinforsbergju) [inline app/src/main/java/com/example/football_manager/News/EditNews.kt]: Remove prints

File: app/src/main/java/com/example/football_manager/News/ViewAllNews.kt [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+                ) {
+                    Text(
+                        text = "Create News",
+                        fontSize = 20.sp,
+                        color = Color.White
+                    )
+                }
+            }
+
+            val startIndex = (currentPage - 1) * itemsPerPage
```
Comment (albinforsbergju) [inline app/src/main/java/com/example/football_manager/News/ViewAllNews.kt]: I think it would look better without pagination.

**Review by albinforsbergju — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/football_manager/News/CreateNews.kt (hunk 1 above)
Comment (GabrielAdward) [reply app/src/main/java/com/example/football_manager/News/CreateNews.kt]: 300?

File: app/src/main/java/com/example/football_manager/News/EditNews.kt (hunk 2 above)
Comment (GabrielAdward) [reply app/src/main/java/com/example/football_manager/News/EditNews.kt]: I have specific variables related to the EditNews.kt and to CreateNews.kt. I have done the same for activities and the laboratory assignment. Would Peter react in a bad way or give us less in grade? 
The only reason I ask is to improve my programming skills.

File: app/src/main/java/com/example/football_manager/News/CreateNews.kt (hunk 1 above)
Comment (albinforsbergju) [reply app/src/main/java/com/example/football_manager/News/CreateNews.kt]: Tänk att man ska kunna skriva typ en artikel, truppen till match etc. 
Nästan man man inte behöver sätta någon max längd.

File: app/src/main/java/com/example/football_manager/News/EditNews.kt (hunk 2 above)
Comment (albinforsbergju) [reply app/src/main/java/com/example/football_manager/News/EditNews.kt]: Tänk DRY. 
Nästan den skulle vara bra att ha en egen fil med alla dessa, oberoende på vilken view.

File: app/src/main/java/com/example/football_manager/News/CreateNews.kt (hunk 1 above)
Comment (GabrielAdward) [reply app/src/main/java/com/example/football_manager/News/CreateNews.kt]: Ok!

File: app/src/main/java/com/example/football_manager/News/EditNews.kt (hunk 2 above)
Comment (GabrielAdward) [reply app/src/main/java/com/example/football_manager/News/EditNews.kt]: Kör globala variabler

File: app/src/main/java/com/example/football_manager/News/ViewAllNews.kt [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+                    color = Color.White
+                )
+            }
+        }
+
+        sortedList.forEach { news ->
+            Column(
+                modifier = Modifier
+                    .padding(vertical = 15.dp)
+                    .clickable { navController.navigate("ViewOne/${news.id}") }
```
Comment (albinforsbergju) [inline app/src/main/java/com/example/football_manager/News/ViewAllNews.kt]: This leads to the activities view one?

File: app/src/main/java/com/example/football_manager/News/ViewOneNews.kt [hunk 6]
```diff
@@ -0,0 +1,109 @@
+package com.example.football_manager.News
```
Comment (albinforsbergju) [inline app/src/main/java/com/example/football_manager/News/ViewOneNews.kt:1]: Looks good, can't test without completing the first comment.

**Review by albinforsbergju — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/football_manager/News/ViewAllNews.kt (hunk 5 above)
Comment (GabrielAdward) [reply app/src/main/java/com/example/football_manager/News/ViewAllNews.kt]: Changed it to ViewOneNews

File: app/src/main/java/com/example/football_manager/News/ViewOneNews.kt (hunk 6 above)
Comment (GabrielAdward) [reply app/src/main/java/com/example/football_manager/News/ViewOneNews.kt:1]: First comment done

**Review by albinforsbergju — APPROVED** [review]
LGTM!

**Pull request merged**